test/test_fx.py

```diff
@@ -0,0 +1,61 @@
+import torch
+from torch.fx import GraphModule
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+class TestFX(TestCase):
```

Comment:
does scripting of a GraphModule work or not yet? (curious whether `inspect` works on generated forward)

Reply:
It does work, see `eval_with_source` for how we inject the source information in a way such that inspect can find it.